language/transaction-builder/generator/tests/generation.rs

```diff
@@ -274,3 +274,65 @@ fn test_that_java_code_compiles_and_demo_runs() {
     );
     assert!(output.status.success());
 }
+
+#[test]
+#[ignore]
+fn test_that_golang_code_compiles_and_demo_runs() {
+    let registry = get_libra_registry();
+    let abis = get_stdlib_script_abis();
+    let dir = tempdir().unwrap();
+
+    let config = serdegen::CodeGeneratorConfig::new("libra_types".to_string());
```

Comment:
libra_types should be libratypes
no underscore or dash in package name see https://blog.golang.org/package-names#TOC_2.